app.rb

```diff
@@ -47,7 +47,7 @@ def openid_configuration
 
   def token(code)
     json HTTParty.post(
-      openid_configuration[:token_endpoint],
+      ENV['IDP_TOKEN_URL'] || openid_configuration[:token_endpoint],
```

Comment:
Yeah, thats kinda what i was thinking. Im not sure if there is a way to...keep the auth stuff from the initial request. 

Any ideas @pkarman ?